src/aiogram_dialog/widgets/kbd/request.py

```diff
@@ -34,11 +40,99 @@ async def _render_keyboard(
         ]
 
 
+class RequestUsers(Keyboard):
+    def __init__(
+        self,
+        text: Text,
+        request_id: int,
```

Comment:
Why not generation request id?

I'd expect that request for old dialog is not processed by new on